sdk/spring/spring-cloud-azure-starter-monitor-test/src/test/java/com/azure/SpringMonitorTest.java

```diff
@@ -41,6 +41,7 @@
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Disabled
```

Comment:
Why is this test disabled?

Reply:
Jean asked me to disable it.. according to him.. he will update it in upstream.